### PR TITLE
added wrn-fix-it bash variant

### PR DESCRIPTION
**Pull request by itsjeffersonli**
Added wrn-fix-it bash that is compatible with linux and mac os

### CODE REVIEW

File: wrn-fix-it.sh [hunk 1]
```diff
@@ -0,0 +1,269 @@
...
+# just donate to keep this project alive.
+#
+# Buy Me A Coffee ............ [1]
+# PayPal ..................... [2]
+# Back to Main Menu .......... [3] (enter)
+#
+"
+    read -p "# $ WRN Fix IT> " donate_choice
+    if [ $donate_choice == "1" ]; then
+        xdg-open "https://www.buymeacoffee.com/warengonzaga"
```
Comment (WarenGonzaga) [inline wrn-fix-it.sh:231]: Please use this https://wrngnz.ga/bmc

File: wrn-fix-it.sh [hunk 2]
```diff
@@ -0,0 +1,269 @@
...
+    elif [ $donate_choice == "2" ]; then
+        xdg-open "https://www.paypal.com/paypalme/warengonzagaofficial"
```
Comment (WarenGonzaga) [inline wrn-fix-it.sh:233]: Use this instead https://wrngnz.ga/paypal

**Review by WarenGonzaga — CHANGES_REQUESTED** [review]
Just change the URLs mentioned and this is good to go!

**Comment by itsjeffersonli** [comment]
Just a moment let me change the links to your specifications

**Comment by itsjeffersonli** [comment]
The links are already changed to your specifications

**Review by WarenGonzaga — APPROVED** [review]
All good! Let's merge this to the dev branch!

**Pull request merged**